packages/web-components/src/global/colors.css

```diff
@@ -0,0 +1,265 @@
+  --ic-theme-primary-a: var(--ic-theme-blue-primary-a);
+  --ic-theme-primary: rgb(
+    var(--ic-theme-primary-r) var(--ic-theme-primary-g)
+      var(--ic-theme-primary-b) / var(--ic-theme-primary-a)
+  );
+  --ic-theme-secondary: rgb(
+    calc(var(--ic-theme-primary-r) * 0.8) calc(var(--ic-theme-primary-g) * 0.8)
+      calc(var(--ic-theme-primary-b) * 0.8) / var(--ic-theme-primary-a)
+  );
+  --ic-theme-tertiary: rgb(
+    calc(var(--ic-theme-primary-r) * 0.6) calc(var(--ic-theme-primary-g) * 0.6)
+      calc(var(--ic-theme-primary-b) * 0.6) / var(--ic-theme-primary-a)
+  );
+  --ic-theme-secondary-light: rgb(
+    calc(var(--ic-theme-primary-r) + ((255 - var(--ic-theme-primary-r)) * 0.2))
+      calc(
+        var(--ic-theme-primary-g) + ((255 - var(--ic-theme-primary-g)) * 0.2)
+      )
+      calc(
+        var(--ic-theme-primary-b) + ((255 - var(--ic-theme-primary-b)) * 0.2)
+      ) / var(--ic-theme-primary-a)
+  );
+  --ic-theme-tertiary-light: rgb(
+    calc(var(--ic-theme-primary-r) + ((255 - var(--ic-theme-primary-r)) * 0.4))
+      calc(
+        var(--ic-theme-primary-g) + ((255 - var(--ic-theme-primary-g)) * 0.4)
+      )
+      calc(
+        var(--ic-theme-primary-b) + ((255 - var(--ic-theme-primary-b)) * 0.4)
+      ) / var(--ic-theme-primary-a)
+  );
+
+  /* ic-theme-calc and ic-theme-text returns if black or white font colors should be used for color contrast reasons by:
+    - Multiplying each RGB value by a set number: https://www.w3.org/TR/AERT/#color-contrast
+    - Adding them together and dividing by 1000
+    - Calculating this for the black and white text RGB values, adding them together and dividing by 2 gives the threshold for determining the font color
+    - Due to no if statement in CSS, high RGB values are capped at 255 and low RGB capped at 0
+    - Subtracting the dark mode threshold from the calculation and multiplying by -10000 returns a large positive or negative number
+    - The RGB function caps negative numbers to RGB(0, 0, 0) or positive numbers to RGB(255, 255, 255)
+    - The ICDS does not use pure black RGB(0, 0, 0) as it's font, it uses #0b0c0c/RGB(11, 12, 12)
+    - The max() CSS function therefore ensures any negative numbers can be no less than 0
+    - As any negative numbers are now 0, finally the numbers 11, 12 and 12 are added to the R, G, B values to make the ICDS shade of black
+    - A similar adjustment is made to add 60, 70 and 77 to give hover & active colors
+    - The min() CSS function is then used to ensure that none of the values exceeds 255
+    - This is due to an issue found in Chromium v117 (sept 2023), where very large values would result the alpha value being ignored and everything appearing white
+    - The result then returns the RGB values for the ICDS white or black font color, dependent on which is the most color contrast compliant
+    This is a similar calculation to it's JS counterpart: "getThemeFontColor"
+  */
+  --ic-theme-dark-mode-threshold: 133.3505;
```

Comment:
Would we actually want some of the old theme (soon to be called brand) variables duplicated? As regardless of the theme (light or dark), the brand colours would remain the same, as would the calculations throughout. Or is it a case we need them for it to work?

Reply:
as discussed, these are not duplicated